doc/architecture/platforms/virtual.rst

```diff
@@ -0,0 +1,12 @@
+Insecure Virtual
+===================
+
+How to use the Insecure Virtual Platform
+----------------------
+The insecure virtual platform can run on any hardware supported by CCF.
+
+To use the insecure virtual platform, ``virtual`` must be specified in ``COMPILE_TARGETS`` at compile time. If any other targets are specified, you will also need to specify the ``TEST_ENCLAVE=virtual`` runtime argument.
```

Comment:
`TEST_ENCLAVE=virtual` is only a test, Python-infra, feature and is part of the CCF release.

Reply:
Is that because a CCF release will only have one build in?

Reply:
`TEST_ENCLAVE` is only used by our infra `e2e_args` as a shorthand to the `--enclave-type` argument and which can easily be passed to `ctest`.

Reply:
Makes sense, I was asking about builds in releases since if there isn't a way to select build at runtime, I assume releases only contain one?

Reply:
Releases currently contain both the release SGX and debug SGX enclaves, as well as the virtual one. The `enclave.type` also needs to be specified in the node configuration file (see https://microsoft.github.io/CCF/main/operations/configuration.html#enclave).

Reply:
https://github.com/microsoft/CCF/pull/4330/commits/1263baedcfab78e5488ff8d47c5da46581eb5508